stacks/cli/src/commands/login.ts

```diff
@@ -7,7 +7,9 @@ import {
   validators
 } from "@ionic/cli-framework";
 import {Command} from "../interface";
-import * as loginService from "../login.service";
+import * as authenticationService from "../authentication.service";
+import * as fileService from "../file.service";
```

Comment:
@htuna07 services are more related with remote services on the web. That's why fileService doesn't seem to be proper naming for me. We might use something of io or something. @thesayyn Is there a common naming convention for io classes in the project? I've seen StdOut in the functions module so we might follow the same naming convention.